packages/components/src/Page/Page.css

```diff
@@ -78,3 +78,16 @@
 .titleBar.small .actionButton + .actionButton {
   margin-left: var(--space-small);
 }
+
+.titleBar .subtitle {
```

Comment:
Right now this is giving us no margin at the smallest size, and too much space on the medium-up sizes:

I think part of the issue is the [parent `<Content>` element ](https://github.com/GetJobber/atlantis/pull/457/files#diff-0a99e41ee8928670c3b01e35510e5adbe2e64d0d51dab50d7eb059e602eccc20R109) being set to large.

If we remove the "large" on that element, I think we can actually get rid of all these checks and just go

```
.titleBar .subtitle {
  margin-bottom: var(--space-base);
}
```

And get pretty even spacing the whole way down (I just gathered screenshots and pasted them in Figma...):
![image](https://user-images.githubusercontent.com/39704901/105784781-aff6ec80-5f36-11eb-8d6a-640ff8549c21.png)